remixer_core/src/main/java/com/google/android/libraries/remixer/Remixer.java

```diff
@@ -132,6 +135,34 @@ public void addItem(RemixerItem remixerItem) {
     return list;
   }
 
+  /**
+   * Notifies all other instances of variables with the same key that the value has changed and
+   * informs the synchronization mechanism as well.
+   */
+  void onValueChanged(Variable variable) {
+    synchronizationMechanism.onValueChanged(variable);
+    List<RemixerItem> itemList = getItemsWithKey(variable.getKey());
+    for (RemixerItem item : itemList) {
+      if (item != variable) {
+        ((Variable) item).setValueWithoutNotifyingOthers(variable.getSelectedValue());
```

Comment:
seems like Remixer is still responsible for triggering other variables when one changes. Any way the synchronizationMechanism can just take care of this?